### DISCUSSION TITLE
thinko/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#5] [Announcements]
# thinko/

Shader Objects

https://www.supergoodcode.com/thinko/

**Comment by mbriar** [comment]
> which is the only reason I have 300+ hours logged in the benchmark mode of Tomb Raider (2013

Sadly the benchmark is not representative of the actual in-game performance at all in this game, which is pretty terrible on every driver for the Linux port. Any actual video-game-players are probably much better off with the Windows version on Proton still, even if you get some stutters due to DXVK+GPL not covering tessellation shaders.

**Reply by zmike** [reply]
Good thing we only care about benchmarks here

**Comment by jams3223** [comment]
Is the secret related to lighting and starts with ray ? ;)

**Reply by zmike** [reply]
I couldn't possibly no comment

**Comment by mrdeathjr28** [comment]
In this mesa commit

zink: disable EXT_extended_dynamic_state2 on AMDPRO

appear this:

this is broken beyond space and time in 22.10-1395274

LMFAO

**Comment by kklem0** [comment]
Hmmm.. .Where is my 1000% upgrade this week?
